### PR TITLE
[OFE-37] - Website Accessibility 

### PR DESCRIPTION
**Pull request by nashthecoder**
[OFE-37] - Website Accessibility 
  - added alt tags on images.
  - added labels on form control on contact form
  - added aria labels on hidden and visible elements without titles. 
  - increaser contracts on cookie banner by changing colors of text and background. 

[OFE-37]: https://triggerise.atlassian.net/browse/OFE-37

### CODE REVIEW

File: _sass/_blocks.scss [hunk 1]
```diff
@@ -2739,10 +2742,11 @@ blockquote:after {
   }
 
   p {
-    color: $pink;
+    color: #ffffff;
```
Comment (a-bresler) [inline _sass/_blocks.scss]: Is there no `$white` variable or should we create one for white? 

Same applies to the blue (#1D214F) used above - if it's a commonly used colour used throughout the website?

File: _includes/contact.html [hunk 2]
```diff
@@ -3,12 +3,15 @@
...
         <h2>Let's talk</h2>
         <form action="https://getform.io/f/ebe28e33-81f6-4ba8-8dd2-789e88f59ea5" method="POST">
-            <label>Name</label>
-            <input type="text" name="name">
-            <label>Email address</label>
-            <input type="email" name="email">
-            <label>Message</label>
-            <textarea type="text" name="message"></textarea>
+            <label>Name
+                <input type="text" name="name">
```
Comment (a-bresler) [inline _includes/contact.html:7]: 👌

File: _sass/_blocks.scss (hunk 1 above)
Comment (nashthecoder) [reply _sass/_blocks.scss]: @a-bresler let me look into this.

**Review by a-bresler — APPROVED** [review]
Looks good!

Just a comment on storing commonly used colours in a readable variable :)

giphy (1)

File: _sass/_blocks.scss (hunk 1 above)
Comment (nashthecoder) [reply _sass/_blocks.scss]: @a-bresler  done. Thank you for the feedback.

Comment (a-bresler) [reply _sass/_blocks.scss]: giphy

**Comment by nashthecoder** [comment]
The gifs @a-bresler are epic!

**Pull request merged**